src/psfmachine/tpf.py

```diff
@@ -420,6 +413,7 @@ def from_TPFs(
         magnitude_limit=18,
         dr=2,
         time_mask=None,
+        do_focus_mask=True,
```

Comment:
new kwarg for `from_TPFs`

Reply:
can we have something like
`apply_focus_mask`?

Reply:
This keyword also needs to be in the `__init__` function, right now it's just in the `from_TPFs` static method.

Reply:
So I thought that `from_TPFs` will create the mask, but the actual masking happens in the `__init__` depending if `time_mask` and `focus_map` are valid or `None`